types/testing-library__react/index.d.ts

```diff
@@ -1,15 +1,16 @@
-// Type definitions for @testing-library/react 9.1
+// Type definitions for @testing-library/react 10.0
 // Project: https://github.com/testing-library/react-testing-library
 // Definitions by: Alex Krolick <https://github.com/alexkrolick>
 //                 Kent C Dodds <https://github.com/kentcdodds>
 //                 Sebastian Silbermann <https://github.com/eps1lon>
 //                 Weyert de Boer <https://github.com/weyert>
 //                 Ifiok Jr. <https://github.com/ifiokjr>
 //                 Daniel Afonso <https://github.com/danieljcafonso>
+//                 Jed Mao <https://github.com/jedmao>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 3.0
+// TypeScript Version: 3.8
```

Comment:
Could you explain why we need `import type`? As far as I know this is only useful for isolatedModules and babel. We need a stronger argument for leaving older TS versions behind.